### ISSUE TITLE
Navbar's elements are not accessible by keyboard in mobile view

### ISSUE THREAD

**Issue by hannyle** [CSCfi/swift-browser-ui#890]
#### Describe the bug

In SD Connect's browser page, navbar's elements are not accessible in smaller screens.

#### To Reproduce
Steps to reproduce the behaviour:
1. Go to SD Connect's browser page
2. Shrink the SD Connect's browser page to smaller size until the navbar collapes and is replaced with a _hamburger_ icon.
3. Try using the Tab key to open elements inside the _hamburger_icon

#### Expected behavior

Elements under Navbar's menu are accessible by keyboard in small screens

#### Screenshots
![Screenshot 2022-12-14 at 18 16 33](https://user-images.githubusercontent.com/23532204/207649673-4cd97384-335b-4871-abea-7e2dfbc41333.png)
